torch/csrc/jit/script/module.h

```diff
@@ -74,6 +74,10 @@ struct Method {
     get_executor().run(stack);
   }
 
+  void run(Stack&& stack) {
```

Comment:
I don't think we want to add this method. TorchScript has almost no externally visible side effects, so you really only want to run it with a stack that will survive this call, so you can read out its outputs.